### PR TITLE
feat: enable add web part feature to SPFx projects for v3

### PR DESCRIPTION
**Pull request by HuihuiWu-Microsoft**
E2E TEST: will add later
User experience:
addwebpart

### CODE REVIEW

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -929,6 +935,10 @@
   "driver.spfx.warn.noTenantAppCatalogFound": "No tenant app catalog found, retry: %s",
   "driver.spfx.error.failedToGetAppCatalog": "Cannot get app catalog site url after creation. You may need wait a few minutes and retry.",
   "driver.spfx.error.noValidAppCatelog": "There is no valid app catalog in your tenant. You can update the property 'createAppCatalogIfNotExist' in %s to true if you want Teams Toolkit to create it for you or you can create it by yourself.",
+  "driver.spfx.add.description": "add additional web part to SPFx project",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:938]: Capitalize 'a'

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -929,6 +935,10 @@
...
+  "driver.spfx.add.successNotice": "Web part %s successfully added to project.",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:939]: -> was successfully

File: packages/fx-core/resource/package.nls.json (hunk 1 above)
Comment (HuihuiWu-Microsoft) [reply packages/fx-core/resource/package.nls.json:938]: This is as designed by unified output of drivers, all driver description are not capitalized. e.g. https://github.com/OfficeDev/TeamsFx/blob/1aadc352afab2cd34052596bcb377e1fcf634c9a/packages/fx-core/resource/package.nls.json#L917

**Review by sffamily — APPROVED** [review]

File: packages/vscode-extension/package.nls.json [hunk 3]
```diff
@@ -92,6 +92,8 @@
     "teamstoolkit.commands.validateManifest.title": "Validate Manifest File",
     "teamstoolkit.commands.viewSamples.title": "View Samples",
     "teamstoolkit.commands.zipPackage.title": "Zip Teams App Package",
+    "teamstoolkit.commmands.addWebpart.title": "Add SPFx web part",
```
Comment (tecton) [inline packages/vscode-extension/package.nls.json:95]: This should be capitalized, "Add SPFx Web Part"

Comment (HuihuiWu-Microsoft) [reply packages/vscode-extension/package.nls.json:95]: Updated

**Review by tecton — DISMISSED** [review]

**Review by jayzhang — DISMISSED** [review]

**Review by tecton — APPROVED** [review]

**Review by sffamily — DISMISSED** [review]

**Comment by HuihuiWu-Microsoft** [comment]
Code coverage increased initially (https://app.codecov.io/github/OfficeDev/TeamsFx/commit/169e4375703de9214e34e79a18c9ce50fd3e4459), the decrease is due to merge with latest dev branch which has coverage loss.

**Pull request merged**